### PR TITLE
SpringBoot Service Template For Eclipse

### PR DESCRIPTION
**Pull request by GeeseHoward**
Eclipse project for development of backend services.
Contains Sonarqube configuration.

### CODE REVIEW

**Review by bonellon — DISMISSED** [review]

**Review by ats0stv — CHANGES_REQUESTED** [review]
Can you please remove off the /bin/* files from the commit?

**Comment by bonellon** [comment]
Should we put this under PoC instead of Code/ ?

File: Code/SpringBootProjectTemplate/build.gradle [hunk 1]
```diff
@@ -0,0 +1,20 @@
+plugins {
+	id 'org.springframework.boot' version '2.1.3.RELEASE'
+	id 'java'
+	id "org.sonarqube" version "2.7"
+}
+
+apply plugin: 'io.spring.dependency-management'
+
+group = 'com.tutorialspoint'
```
Comment (ats0stv) [inline Code/SpringBootProjectTemplate/build.gradle]: Package names as ie.tcd.* will be good

File: Code/SpringBootProjectTemplate/gradle.properties [hunk 2]
```diff
@@ -0,0 +1,4 @@
+systemProp.sonar.host.url=http://35.242.176.105:80
+ 
+#----- Token generated from an account with 'publish analysis' permission
+systemProp.sonar.login=9d0603ce578cc3722a7511bbc7307976218481de
```
Comment (ats0stv) [inline Code/SpringBootProjectTemplate/gradle.properties]: New line at the end

**Review by ats0stv — CHANGES_REQUESTED** [review]

**Comment by GeeseHoward** [comment]
> Can you please remove off the /bin/* files from the commit?

Done

**Comment by GeeseHoward** [comment]
> Should we put this under PoC instead of Code/ ?

Done

File: Code/SpringBootProjectTemplate/build.gradle (hunk 1 above)
Comment (GeeseHoward) [reply Code/SpringBootProjectTemplate/build.gradle]: Done

File: Code/SpringBootProjectTemplate/gradle.properties (hunk 2 above)
Comment (GeeseHoward) [reply Code/SpringBootProjectTemplate/gradle.properties]: Done

File: PoC/SpringBootProjectTemplate/src/main/java/com/devglan/SpringBootProjectTemplateApplication.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package com.devglan;
```
Comment (ats0stv) [inline PoC/SpringBootProjectTemplate/src/main/java/com/devglan/SpringBootProjectTemplateApplication.java]: These package names too can be changed to match our repo
- ie.tcd.springtemplate.*

File: PoC/SpringBootProjectTemplate/src/test/java/com/devglan/SpringBootProjectTemplateApplicationTests.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
+package com.devglan;
```
Comment (ats0stv) [inline PoC/SpringBootProjectTemplate/src/test/java/com/devglan/SpringBootProjectTemplateApplicationTests.java]: Same here

**Review by ats0stv — CHANGES_REQUESTED** [review]

File: PoC/SpringBootProjectTemplate/src/main/java/com/devglan/SpringBootProjectTemplateApplication.java (hunk 3 above)
Comment (GeeseHoward) [reply PoC/SpringBootProjectTemplate/src/main/java/com/devglan/SpringBootProjectTemplateApplication.java]: Done

File: PoC/SpringBootProjectTemplate/src/test/java/com/devglan/SpringBootProjectTemplateApplicationTests.java (hunk 4 above)
Comment (GeeseHoward) [reply PoC/SpringBootProjectTemplate/src/test/java/com/devglan/SpringBootProjectTemplateApplicationTests.java]: Done

**Review by ats0stv — APPROVED** [review]

**Review by bonellon — APPROVED** [review]

**Pull request merged**